server/service/src/standard_reports.rs

```diff
@@ -0,0 +1,88 @@
+use repository::{
+    ContextType, FormSchemaJson, FormSchemaRowRepository, ReportRow, ReportRowRepository,
+    StorageConnection, StorageConnectionManager,
+};
+use rust_embed::RustEmbed;
+use thiserror::Error;
+
+use crate::report::definition::ReportDefinition;
+use log::info;
+use serde::{Deserialize, Serialize};
+
+#[derive(RustEmbed)]
+// Relative to server/Cargo.toml
+#[folder = "../reports/generated"]
+
+pub struct EmbeddedStandardReports;
+
+#[derive(Debug, Error)]
+#[error("No standard reports found")]
+pub struct StandardReportsError;
+
+#[derive(Clone)]
+pub struct StandardReports;
+
+impl StandardReports {
+    // Load embedded reports
+    pub fn load_reports(connection_manager: StorageConnectionManager) -> Result<(), anyhow::Error> {
+        let con = connection_manager.connection()?;
+
+        for file in EmbeddedStandardReports::iter() {
+            if let Some(content) = EmbeddedStandardReports::get(&file) {
+                let json_data = content.data;
+                let reports_data: ReportsData = serde_json::from_slice(&json_data)?;
+                let _ = StandardReports::upsert_reports(reports_data, &con)?;
+            }
+        }
+        Ok(())
+    }
+
+    pub fn upsert_reports(
+        reports_data: ReportsData,
+        con: &StorageConnection,
+    ) -> Result<(), anyhow::Error> {
+        for report in reports_data.reports {
+            if let Some(form_schema_json) = &report.form_schema {
+                FormSchemaRowRepository::new(con).upsert_one(form_schema_json)?;
+            }
+
+            ReportRowRepository::new(&con).upsert_one(&ReportRow {
+                id: report.id.clone(),
```

Comment:
Upsert is using the id here, I think that's kind of ok given that it includes the name and id,but we shoud probalby look up using the code first then upsert based on the ID we get back, otherwise the codes doing nothing...